terraform/environments/electronic-monitoring-data/modules/lambdas/variables.tf

```diff
@@ -110,4 +110,18 @@ variable "production_dev" {
   type        = string
   nullable    = true
   default     = null
+}
+
+variable "ecr_repo_name" {
+  description = "ECR repo in shared services acc"
+  type = string
+  nullable = false
+  default = "electronic-monitoring-lambda-functions"
+}
+
+variable "function_tag" {
+  description = "Custom tag in ECR repo"
+  type = string
+  nullable = true
+  default = null
```

Comment:
oooh

Reply:
might need to look at this slightly differently, but id hardcoded `electronic-monitoring-data-lambdas` and the function name in

but if were all using our own images in create a data task we could do something similar